### ISSUE TITLE
Any update regarding upgrade to 0.74?

### ISSUE THREAD

**Issue by neeteshraj** [thecodingmachine/react-native-boilerplate#426]
### Describe the solution you'd like

Any update regarding upgrade to React Native 0.74?

**Comment by ha-family** [comment]
I'm also eagerly waiting for it! ~

**Comment by JeremyDolle** [comment]
Hello, it will come soon, I am working on it with some fixs and improvements on Theme generation

**Comment by ha-family** [comment]
> Hello, it will come soon, I am working on it with some fixs and improvements on Theme generation

The last time I used this boilerplate was version 3.5.0, which used React Native version 0.69. The biggest problem I encountered at that time was with the useTheme hook. Every time it was called, it would merge files, which significantly impacted rendering performance when used in a list with many components calling useTheme.

I'm not sure if this issue has been fixed in the latest version, but I wanted to bring it up just in case the same problem occurs in the new version.
